htdocs/class/xoopslocal.php

```
@@ -58,20 +58,12 @@ public static function substr($str, $start, $length, $trimmarker = '...')
-        if (defined('XOOPS_USE_MULTIBYTES') && 1 === (int)XOOPS_USE_MULTIBYTES) {
-            if (function_exists('mb_convert_encoding')) {
-                $converted_text = mb_convert_encoding($text, 'UTF-8', 'auto');
-                if ($converted_text !== false && !is_array($converted_text)) {
-                    return $converted_text;
-                } else {
-                    // Handle the failure case, maybe log an error or return the original text
-                    return $text;
-                }
-
-            }
+        $converted_text = mb_convert_encoding($text, 'UTF-8', 'auto');
```

PHPMD (2.15.0)

[error] The variable $converted_text is not named in camelCase. (undefined)

(CamelCaseVariableName)